be/src/exec/vectorized/join_hash_map.h

```diff
@@ -65,8 +65,9 @@ enum class JoinHashMapType {
 enum class JoinMatchFlag { NORMAL, ALL_NOT_MATCH, ALL_MATCH_ONE, MOST_MATCH_ONE };
 
 struct JoinKeyDesc {
-    PrimitiveType type;
+    const TypeDescriptor* type = nullptr;
```

Comment:
why we change PrimitiveType to TypeDescriptor.

Reply:
Used to create key column,if it's not ColumnRef